tools/d2g/d2g.go

```diff
@@ -357,10 +357,10 @@ func runCommand(containerName string, dwPayload *dockerworker.DockerWorkerPayloa
 		command.WriteString(" --security-opt=seccomp=unconfined")
 	}
 	command.WriteString(createVolumeMountsString(dwPayload, wdcs))
+	command.WriteString(envMappings(dwPayload))
```

Comment:
Any reason for the different ordering?

Reply:
No particular reason.

Reply:
Reordered.

Reply:
It'll make it so you don't have to reorder so many of the test cases, keep the diff to what's truly changing here 👍🏻